daq/topology.py

```diff
@@ -519,6 +534,10 @@ def _generate_port_acl(self, port):
         target_mac = None
         rules = []
 
+        if port != self.sec_port:
```

Comment:
why is this check necessary?

Reply:
Since the ACL is applied to the sec_port, no point trying to add a rule which diverts it back.

Reply:
But this function should never be called with port == self.sec_port
If it is, then that's the core issue.  If you're not sure, then add an assert to verify.

Reply:
Changed.